jest.preset.js

```diff
@@ -1,3 +1,11 @@
 const nxPreset = require('@nx/jest/preset').default;
-
-module.exports = { ...nxPreset };
+module.exports = {
+  ...nxPreset,
+  testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
+  transform: {
+    '^.+\\.(ts|js|html)$': 'ts-jest',
+  },
+  resolver: '@nx/jest/plugins/resolver',
+  moduleFileExtensions: ['ts', 'js', 'html'],
```

Comment:
don't know much about this, but do we need to include jsx and tsx?

Reply:
I guess yes, some of the old apps are still written in js... e.g. registry